js/src/logger.ts

```diff
@@ -4703,6 +4707,80 @@ export type PromptRowWithId<
     ? Pick<PromptRow, "_xact_id">
     : Partial<Pick<PromptRow, "_xact_id">>);
 
+export function deserializePlainStringAsJSON(s: string) {
```

Comment:
If this function isn't used anywhere else we don't need to export it.

Reply:
Kept the export but actually used it